lib/rest/RequestHandler.js

```diff
@@ -404,6 +404,8 @@ class RequestHandler {
                 method += "_OLD";
             }
             route = method + route;
+        } else if(method === "GET" && /\/guilds\/\d+\/channels$/.test(route)) {
```

Comment:
Constructing this regex over and over could be problematic

Reply:
Apparently faster than match() on a regex constant on Node 16